### ISSUE TITLE
Using Rpi3 script on RPi Zero

### ISSUE THREAD

**Issue by SteveBz** [rlancaste/AstroPi3#30]
I'm trying to run your fantastic RPi3 script on a RPi Zero Raspian (just INDI, no PHD2 or viewer).  It's better to set:

-j $(expr $(nproc) + 2)

as 

-j 1

Thoughout, otherwise stuff hangs.  I'l drop other comments as they arise.

Kind regards,

Steve.

**Comment by SteveBz** [comment]
So indiserver works nicely not all of the drivers seem to compile but I'm working on that.  The biggest issue for me is that I can't get ININDWebManagerApp to work.  Here's the error:

CMake Warning (dev) at /usr/share/ECM/modules/ECMFindModuleHelpers.cmake:113 (message):
  Your project should require at least CMake 3.16.0 to use FindKF5.cmake
Call Stack (most recent call first):
  /usr/share/ECM/find-modules/FindKF5.cmake:30 (ecm_find_package_version_check)
  CMakeLists.txt:59 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find KF5DocTools (missing: KF5DocTools_DIR)
-- Could NOT find KF5DocTools: found neither KF5DocToolsConfig.cmake nor kf5doctools-config.cmake
CMake Error at /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find KF5 (missing: DocTools) (found version "5.103.0")
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/ECM/find-modules/FindKF5.cmake:93 (find_package_handle_standard_args)
  CMakeLists.txt:59 (find_package)

Kind regards

Steve.

**Comment by rlancaste** [comment]
It looks like you don't have QT5 and KF5.  It is true that most of KDE has upgrade to QT6 and KF6, but KStars and related problems are still on QT5.  INDI would not be affected by that since it is not based on QT.  I probably need to update the INDIWebManagerApp to make it QT6 compatible.  But if it did not work for this program, it should not work for kstars either.  Was that able to build?

**Comment by SteveBz** [comment]
> It looks like you don't have QT5 and KF5. It is true that most of KDE has upgrade to QT6 and KF6, but KStars and related problems are still on QT5. INDI would not be affected by that since it is not based on QT. I probably need to update the INDIWebManagerApp to make it QT6 compatible. But if it did not work for this program, it should not work for kstars either. Was that able to build?

Ah, I don't know about kstars I didn't test.  I only want indiserver for controlling remote devices and indiserver is working.  I think maybe kstars isn't working either.  I can't find a menu entry and it's not on the $path.  

Thanks

Steve

**Comment by SteveBz** [comment]
Hi Rob,

Well it looks as though I can can install kstars and indi on Ubuntu 24.04 on my main machine and on a RPi0 using apt-get.  Jut not INDIwebmanagerapp.  Is there any workaround I can use?

Kind regards 

Steve.

**Comment by rlancaste** [comment]
You can always start an indiserver manually or with a script if needed

**Comment by SteveBz** [comment]
Yes, that's what I'm doing. The only issue seems to be that I have to start kstars to connect. After connection, I can then stop kstars and it continues connecting.  I'm not clear why I need to do this. I'm just streaming weather data.

**Comment by rlancaste** [comment]
I don't know that much about that driver, but my guess would be that when you connect to it, some initialization stuff happens during the connection and starts your streaming.  You could build and install the indiwebmanager app.  Or another option might be to just install indiweb and have a script start it up.  Once indiweb is running, you can always start and stop the drivers from the web.
